peepingtom/datablocks/simpleblocks/orientationblock.py

```diff
@@ -69,5 +69,10 @@ def zyx_vectors(self):
             all_vectors[ax] = (self.oriented_vectors(ax).sel(spatial=list(axes)))
         return all_vectors
 
+    @property
+    def zyx(self):
+        zyx = list('xyz')
+        return self.data.sel(spatial=zyx, spatial2=zyx)
```

Comment:
what's going on here? should it be 'zyx' in the zyx variable?

Reply:
You need to provide a list with the oorder of coordinates. Making the list and then passing the variable was less clunky and more legible than `['z', 'y', 'x']`.